test/livebook_web/live/home_live_test.exs

```diff
@@ -325,6 +325,25 @@ defmodule LivebookWeb.HomeLiveTest do
     end
   end
 
+  describe "public open endpoint" do
+    test "checkouts the directory when a directory is passed", %{conn: conn} do
+      directory_path = Path.join(File.cwd!(), "test")
+
+      {:ok, view, _} = live(conn, "/?path=#{directory_path}")
+
+      assert render(view) =~ directory_path
+    end
+
+    @tag :tmp_dir
+    test "opens a file when livebook file is passed", %{conn: conn, tmp_dir: tmp_dir} do
+      notebook_path = Path.join(tmp_dir, "notebook.livemd")
+
+      {:ok, view, _} = live(conn, "/open?path=#{notebook_path}")
+
+      assert render(view) =~ "My notebook"
```

Comment:
I think here you're still on the homepage and it's just that `My notebook` is created in another test and it's listed on the page :D

We need yo write contents to the test notebook like `# Notebook to open` and then assert against that.

The view most likely has an error saying the file cannot be open because it doesn't exist, since we never create it :)